src/common/darktable.c

```diff
@@ -413,6 +414,14 @@ int dt_init(int argc, char *argv[], const int init_gui, lua_State *L)
   }
 #endif
 
+  if(!dt_openmp_init_stacksize())
+  {
+    fprintf(stderr, "[dt_init] your system uses a low default stacksize and we could not set\n");
+    fprintf(stderr, "[dt_init] the OMP_STACKSIZE environment variable to %dB.\n", SAFESTACKSIZE);
+    fprintf(stderr, "[dt_init] Please set the OMP_STACKSIZE environment variable before starting darktable\n");
+    return 1;
+  }
+
```

Comment:
I'll return to my original idea of using the ulimit softlimit as the default stacksize. If darktable segfaults because that's to low then users can change it by increasing that. The problem with musl is that they have a hardcoded 80K stack size.